### PR TITLE
add Chinese deploy parachain content

### PR DESCRIPTION
**Pull request by ansonla3**

### CODE REVIEW

File: docs/lang/chinese.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 [Governance - 治理](https://mp.weixin.qq.com/s/yyPaHsU7Nf7sUBVGIq1jbw)
 
+
+## Prachain 平行链资料
```
Comment (lsaether) [inline docs/lang/chinese.md]:
```suggestion
## Parachain 平行链资料
```

**Review by lsaether — APPROVED** [review]
just one minor spelling

File: docs/lang/build/deploy-parachains.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+按下 `democracy` -> `propose(proposal,value)` -> `parachains` -> `registerParachain(id,code,initial_head_data)`
+
+在id栏位输入平行链的`id`，在这个简单的adder情况，将会是`100`。在`code`栏位，按下页面按钮之后你可以直接上传你之前编译好的`adder.wasm`二进制档案。在`initial_head_data`，你需要复制和贴下你从校对人取的Hex数据。在`value`栏位，你需要输入最小要求的数值从而创建公投，在Alexander测试网络暂时是5 DOTs。
+
+![registering a parachain](../../img/parachain/register.png)
+
+假如你前往到`Democracy`标签，你应该会看到你的提案在议案部分
+
+一旦你等待的提案成为公投，你可以投票选择`赞成(Nay)`或`反对(Aye)`，投赞成票使你能够部署平行链。
```
Comment (xlc) [inline docs/lang/build/deploy-parachains.md]: 赞成是Nay?
Nay means no Aye means yes

Comment (ansonla3) [reply docs/lang/build/deploy-parachains.md]: yes, good catch

File: docs/lang/build/deploy-parachains.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+现在前往在Polkadot代码库裹的`test-parachains`资料夹并执行生成脚本
+
+```bash
+cd polkadot/test-parachains
+./build.sh
+```
+
+它会在这个资料夹建立简单`adder`平行链的Wasm可执行文件，它将简单地添加发送到给它的消息。Wasm可执行文件会在`parachains/test/res/adder.wasm`路径，所以确保你能在那找到。
+
+你需要生成并运行(Collator)校对人节点从而取得平行链的创世纪状态。
```
Comment (xlc) [inline docs/lang/build/deploy-parachains.md]:
```suggestion
你需要生成并运行校对人(Collator)节点从而取得平行链的创世纪状态。
```
括号一般放后面吧

**Pull request merged**